policy.go

```diff
@@ -158,6 +161,10 @@ func (p *Policy) AllowAttrs(attrNames ...string) *attrPolicyBuilder {
 	return &abp
 }
 
+func (p *Policy) AllowDataAttributes() {
```

Comment:
Needs public documentation comment that would pass golint.

Specifically the comment needs to explain what it will do and why, and then include a warning that the values are not sanitized and that applications that eval() or process the value without having checked and verified the input may be at risk if this option is enabled. This is a 'caveat emptor' option and the person enabling this option needs to fully understand the potential impact with regards to whatever application will be consuming the sanitized HTML afterwards, i.e. if you know you put a link in a data attribute and use that to automatically load some new window... then you're giving the author of a HTML fragment the means to open a malicious destination automatically. Such things only the owner of the web application in question could know, there are enough legitimate uses for this addition, it should just come with a warning..